test/contconic.jl

```diff
@@ -503,5 +503,106 @@ function contconictest(solver::MOI.AbstractSolver, ε=Base.rtoldefault(Float64))
         end
     end
 
+    if MOI.supportsproblem(solver, MOI.ScalarAffineFunction, [(MOI.ScalarAffineFunction{Int}, EqualTo{Int}), (MOI.ScalarAffineFunction{Int}, EqualTo{Rational{Int}}), (MOI.VectorVariablewiseFunction, MOI.PositiveSemidefiniteConeTriangle), (MOI.VectorVariablewiseFunction, MOI.SecondOrderCone)])
```

Comment:
Yes, a solver usually a type that it supports like Float64 and it can add one method that converts addconstraint of any other type to this type. It does not need one for every pair of function/method.